### PR TITLE
Getting started with Linux container security

### PR DESCRIPTION
**Pull request by molly80**
article about linux containers security

### Article Checklist
**NOTE: (1st time contributors - submit ONLY one article to get familiar with the process, quality, and style-once published, you can then submit any number of articles.)**
#### Preliminary Checks - Formatting and Structure 
- [ yes] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [ yes] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [yes ] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [yes ] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [ yes] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [yes ] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [ yes] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [ no] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.
       fixes#1118
#### Grammar Checks
- [ yes] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [yes ] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [yes ] Have you added sources for quotes and images that aren't yours?

#### Technical Checks
- [ ] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution Guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**

### CODE REVIEW

**Comment by molly80** [comment]
This fixes #1118

**Comment by molly80** [comment]
@WanjaMIKE hello, it's not a draft. However, I can make any additions required

**Comment by WanjaMIKE** [comment]
@molly80 did you want to make any additions or its ready?

**Comment by molly80** [comment]
@WanjaMIKE it is ready

**Comment by WanjaMIKE** [comment]
`Okay`

**Comment by hectorkambow** [comment]
plagiarism check done

**Comment by WanjaMIKE** [comment]
@molly80 could you please fix the plagiarism before we continue with the review. Thanks

**Comment by molly80** [comment]
@WanjaMIKE yes sure will send the corrected version later

**Comment by WanjaMIKE** [comment]
@molly80 Still waiting.

**Comment by molly80** [comment]
@WanjaMIKE will send it soon

**Comment by WanjaMIKE** [comment]
@molly80 Unfortunately, if you do not upload the corrected article soon , it will be considered as inactivity and the PR will be closed.

**Comment by molly80** [comment]
Sent could you please check

File: articles/getting-started-with-linux-container-security/index.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# Getting Started With Linux Containers Security
+
+### Overview of containers
+Linux containers, commonly referred to as LXC, are operating systems used to run multiple containers together using a single Linux kernel through a control host. In containers, users can create and manage application containers and systems easily as the API that is sturdy and uses tools that are easy to use.
```
Comment (WanjaMIKE) [inline articles/getting-started-with-linux-container-security/index.md:4]: Could you break this sentence `In containers, users can create and manage application containers and systems easily as the API that is sturdy and uses tools that are easy to use.` into two. In seems difficult to understand.

**Comment by molly80** [comment]
I have corrected please check

**Review by WanjaMIKE — APPROVED** [review]
@molly80 Nice first article. Just a few grammar issues which can be improved next time.

**Pull request merged**